src/style.css

```diff
@@ -0,0 +1,143 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
```

Comment:
You should create separate file for each BEM block. You need folder "blocks" with files stars.css and card.css, and separate files for variables and resets.

Reply:
fixed